examples/network-hhmodel/parameters.rs

```diff
@@ -0,0 +1,28 @@
+use ixa::context::Context;
+use ixa::global_properties::ContextGlobalPropertiesExt;
+use std::fmt::Debug;
+use std::path::Path;
+
+use ixa::define_global_property;
+use ixa::error::IxaError;
+use serde::{Deserialize, Serialize};
+
+#[allow(clippy::module_name_repetitions)]
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct ParametersValues {
+    pub incubation_period: f64,
+    pub infectious_period: f64,
+    pub sar: f64,
+    pub shape: f64,
+    pub infection_duration: f64,
+    pub between_hh_transmission_reduction: f64,
+}
+define_global_property!(Parameters, ParametersValues);
+
+pub fn init(context: &mut Context, file_name: &str) -> Result<(), IxaError> {
```

Comment:
You don't actually need to do this. If you set the property names correctly in the JSON, you can just do `context.load_global_properties()`